test/integration/hipchat_test.exs

```diff
@@ -57,7 +57,7 @@ defmodule Integration.HipChatTest do
     |> with_permission("operable:thorn")
 
     message = send_message user, ~s(@deckard operable:echo "this is a test" | operable:thorn $body)
-    assert_response "þis is a test", after: message
+    assert_response "The thorn command has been deprecated and will be removed in a future release", after: message
```

Comment:
Might be worth moving this to another command since we'll be removing `thorn` soon.